rfc/rfc010-multiple-operators.md

```diff
@@ -0,0 +1,64 @@
+# RFC010 Multiple operator service access
+
+|  |  |
+| :--- | :--- |
+| Nuts foundation | W.M. Slakhorst |
+| Request for Comments: 010 | Nedap |
+|  | January 2021 |
+
+## Multiple operator service access
+### Abstract
+
+This RFC describes an addition to [RFC003](rfc003-oauth2-auithorization.md) and [RFC006](rfc006-distributed-registry.md) on how to handle resource access when a legal party has multiple node operators. This is the case when a care organization uses multiple pieces of software from different SaaS providers. A SaaS provider can alter the care organization DID document so the DID document of the other provider is added as synonym. This is done by using the *also-known-as* relation.
+
+### Status
+
+This document is currently a draft.
+
+### Copyright Notice
+![](../.gitbook/assets/license.png)
+
+This document is released under the [Attribution-ShareAlike 4.0 International \(CC BY-SA 4.0\) license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## 1. Introduction
+
+Care organizations rarely use one piece of software. Larger organizations can use up to a hundred different pieces of software. Every software vendor that has signed a **Data Processing Agreement** with the care organization is allowed to transfer data to any other software vendor within the limits of the **Data Processing Agreement**. 
+
+To support this functionality, some registration and extended validation is required throughout the process of data transfer.
+
+## 2. Terminology
+
+* **Bolt**: a use case built on top of the functionality Nuts provides.
+* **DID**: a decentralized identifier according to the [core-did-spec](https://www.w3.org/TR/did-core). 
+* **DPA**: Data Processing Agreement as defined by the GDPR.
+
+Other terminology is taken from the [Nuts Start Architecture](rfc001-nuts-start-architecture.md#nuts-start-architecture).
+
+## 3. Registration
+
+Each care organization that is registered at a node by a SaaS provider will have its own DID document. If a care organization uses 2 SaaS providers, it'll have two different DID documents. Since identification within the OAuth flow is done by DID, the DID subject need to be aware of each other. This can be done by adding the DID of one document and add it to the other. This is done by using the `alsoKnownAs` field defined in the [core-did-spec](https://www.w3.org/TR/did-core/#also-known-as).
```

Comment:
> This can be done by adding the DID of one document and add it to the other

"adding the DID of one document" already implies implementation, so I'd use "referring" in this sentence, since the implementation is defined in the next sentence (using alsoKnownAs)

This can be done by having the DID documents of the SaaS providers refer to each other.